src/strategy/periodic.rs

```diff
@@ -132,7 +132,7 @@ impl StrategyPeriodic {
             Some(minute_in_week) => {
                 let (weekday, hour, minute) = utils::weekly_minute_as_weekday_time(minute_in_week);
                 format!(
-                    "at {}:{} on {} ({}), subject to time zone caveats.",
+                    "at {}:{0>2} on {} ({}), subject to time zone caveats.",
```

Comment:
```suggestion
                    "at {}:{:0>2} on {} ({}), subject to time zone caveats.",
```
Of course I couldn't make a 3 character PR without a typo -- it should be a 4 character PR.